app/utils.py

```diff
@@ -29,3 +30,54 @@ def parse_phone_number(phone_number):
         stripped = '1' + stripped
     stripped = '+' + stripped
     return stripped
+
+
+def parse_to_db(db, filename):
+    city_default = ', philadelphia, pennsylvania, usa'
```

Comment:
I don't see it in the [documentation here](https://pypi.python.org/pypi/geocoder), so we can definitely use geocoding directly which shouldn't be hard.

My thought was that this parser is only for this particular csv format for the historical data (which would only be from Philly)? Also the viewport is a rectangular bounding box while Philadelphia's city boundaries are oblongish and not rectangular, so there's a robustness tradeoff there too?